includes/class-main.php

```
@@ -209,6 +212,65 @@ public function __construct() {
+		 * When enableLoggedInCache is on, optimisations run even for logged-in users
+		 * so the cached page includes all improvements.
+		 *
+		 * @since 2.8.0
+		 * @return bool True if optimisations may run for the current user.
+		 */
+		private function should_optimise_for_logged_in(): bool {
+			if ( ! is_user_logged_in() ) {
+				return true;
+			}
+			return ! empty( $this->options['cache_settings']['enableLoggedInCache'] ?? false );
+		}
```

PHPMD (2.15.0)

[error] The method should_optimise_for_logged_in is not named in camelCase. (undefined)

(CamelCaseMethodName)